Dockerfile

```
@@ -79,6 +103,77 @@ EXPOSE 8080
+# Switch to use a non-root user from here on
+USER nobody
+
+# Add application
+WORKDIR /var/www/html
+
+# Expose the port nginx is reachable on
+EXPOSE 8080
+
+# Let runit start nginx & php-fpm
+# Ensure /bin/docker-entrypoint.sh is always executed
+ENTRYPOINT ["/bin/docker-entrypoint.sh"]
```

Hadolint

[error] Multiple `ENTRYPOINT` instructions found. If you list more than one `ENTRYPOINT` then only the last `ENTRYPOINT` will take effect

(DL4004)